src/api/RebalancingAPI.ts

```diff
@@ -577,6 +611,44 @@ export class RebalancingAPI {
     );
   }
 
+  private async assertBidWithEther(rebalancingSetTokenAddress: Address, bidQuantity: BigNumber, txOpts: Tx) {
+    this.assert.schema.isValidAddress('rebalancingSetTokenAddress', rebalancingSetTokenAddress);
+    this.assert.common.greaterThanZero(
+      bidQuantity,
+      coreAPIErrors.QUANTITY_NEEDS_TO_BE_POSITIVE(bidQuantity)
+    );
+
+    await this.assert.setToken.isValidSetToken(this.core.coreAddress, rebalancingSetTokenAddress);
+    await this.assert.rebalancing.isInRebalanceState(rebalancingSetTokenAddress);
+    await this.assert.rebalancing.bidAmountLessThanRemainingSets(rebalancingSetTokenAddress, bidQuantity);
```

Comment:
Feels like this assertion should be conditioned on if they're allowing a partialFill or not

Reply:
Yeah agree this is more accurate. Will also update in the bidAsync assertion